### PR TITLE
docs(quickstart): add quickstart docs

### PR DESCRIPTION
**Pull request by iamonuwa**

### CODE REVIEW

File: docs/quickstart/introduction.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: "Introduction"
+description: "Nym connect quickstart"
+hide_title: false
+---
+
+# Introduction
+
+Nym Connect is a one-button interface that is easy to use. With Nym connect, you get 100% privacy on applications you use everyday. Presently, Nym connect has support for telegram, keybase and electrum wallets. This means that you can run sock5 proxy on these applications with Nym connect. More services coming soon.
```
Comment (mfahampshire) [inline docs/quickstart/introduction.md]: remove the line "With Nym connect, you get 100% privacy on applications you use everyday." - we dont have SURBs yet and a few other things so this isn't strictly true

Comment (mfahampshire) [reply docs/quickstart/introduction.md]: Change 'Nym connect has support for telegram, keybase and electrum wallets. This means that you can run sock5 proxy on these applications with Nym connect. More services coming soon.' to something like 'Presently Nym Connect allows you to send traffic for supported applications through the mixnet, shielding your metadata from the wider internet. NymConnect currently offers support for Telegram, Keybase, and the Electrum Bitcoin wallet. More services will be added soon."

File: docs/quickstart/nym-connect.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+sidebar_label: "Nym Connect"
+description: "Getting started with Nym connect"
+hide_title: false
+---
+
+# Nym Connect
+
+Follow this guide to install and run Nym connect.
+
+### Step 1. Download
```
Comment (mfahampshire) [inline docs/quickstart/nym-connect.md]: These steps need a bit of fleshing out - look at the docs for the wallet and copy some of the formatting there :)

File: docs/quickstart/introduction.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+**Nym connect is still in experimental stage. If buggy please send us a [feedback report](https://dyno.gg/form/bbec84eb)**
```
Comment (mfahampshire) [inline docs/quickstart/introduction.md]: change 'if buggy' to something more formal e.g. 'if you experience problems or bugs, please send a feedback from'

File: versioned_docs/version-stable/quickstart/nym-connect.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+### Step 1. Download
+
+Get the latest release of nym connect app [here](https://github.com/nymtech/nym/releases/tag/nym-connect-v1.0.2).
+
+:::note
+Nym connect build for linux currently supports only **debian**. If you wish to run on another distro that is not debian, you'll need to build from source. To build from source, you'll also need a [Developer ID Application](https://developer.apple.com/support/developer-id/).
+:::
+
+#### Building from source
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/nym-connect.md]: Just add a note like those at the top of each of the node binary pages that you need to build the monorepo first, and remove this heading

File: versioned_docs/version-stable/quickstart/nym-connect.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+# Nym Connect
+
+Follow this guide to install and run Nym connect.
+
+### Step 1. Download
+
+Get the latest release of nym connect app [here](https://github.com/nymtech/nym/releases/tag/nym-connect-v1.0.2).
+
+:::note
+Nym connect build for linux currently supports only **debian**. If you wish to run on another distro that is not debian, you'll need to build from source. To build from source, you'll also need a [Developer ID Application](https://developer.apple.com/support/developer-id/).
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/nym-connect.md]: Its not that NymConnect only supports debian, its that the pre-compiled binaries are built for debian-based systems, and if you aren't running on a debian-based flavour of linux you'll need to compile it yourself

Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/nym-connect.md]: You don't need a developer ID - with building the wallet locally you need to comment out some stuff in the config so it doesnt check for a develper token: i imagine we can do something similar here :) 

 see docs/stable/nym-apps/wallet#for-developers

**Review by mfahampshire — CHANGES_REQUESTED** [review]
Overall looking good, I've requested some changes. Look forward to seeing the socks5 section once thats done too

**Comment by mfahampshire** [comment]
A general comment on this as well - you can remove the numbers from the steps, as we don't do that anywhere else. 

I'd say in general look at the formatting and title-ing of the wallet page and work with that as a template

**Comment by iamonuwa** [comment]
Thanks. Resolving these comments.

File: versioned_docs/version-stable/quickstart/introduction.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+### Nym Connect
+
+Nym Connect is a one-button interface that is easy to use. Presently Nym Connect allows you to send traffic for supported applications through the mixnet, shielding your metadata from the wider internet. NymConnect currently offers support for Telegram, Keybase, and the Electrum Bitcoin wallet. More services will be added soon.
+
+**Nym connect is still in experimental stage. If you experience problems or bugs, please send use a feedback [here](https://dyno.gg/form/bbec84eb)**.
+
+### Nym Sock5
+
+Many existing applications are able to use SOCK5 Proxy protocol. With `nym-sock5-client`, applications can bounce their traffic through NYM network. The Nym Sock5 proxy protocol takes a TCP data stream. The stream of data is chopped up into multiple Sphinx packets and numbers are assigned to them.
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/introduction.md]: `NYM` is the token, `Nym` is the network - can you change this please. 

Also typo with `Sock5` needs correcting.

File: versioned_docs/version-stable/quickstart/nym-connect.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+Nym Connect app lets you send traffic through Nym mixnet for supported applications.
+
+You can download it for Mac, Windows or Linux PCs.
+
+[Download Nym Connect](https://github.com/nymtech/nym/releases/tag/nym-connect-v1.0.2)
+
+### Bypassing security warnings
+
+On Windows you will see a security warning pop up when you attempt to run Nym connect. We are in the process of getting app store keys from Microsoft so that this doesn't happen. See the section below for details on steps to bypass these.
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/nym-connect.md]: Is this true for NymConnect as well as the wallet?

File: versioned_docs/version-stable/quickstart/nym-connect.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+git reset --hard # in case you made any changes on your branch
+git pull # in case you've checked it out before
+cd nym/nym-connect # navigate into nym-connect directory
+```
+
+### Removing signing errors when building in development mode
+
+If you're wanting to build Nym connect yourself, you will need to make a few modifications to the file located at `nym-connect/src-tauri/tauri.conf.json` before doing so. These relate to Nym connect being accepted by Mac and Windows app stores, and so aren't relevant to you when building and running Nym connect yourself.
+
+On **all** operating systems:
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/nym-connect.md]: Have you tested / verified this?

File: versioned_docs/version-stable/quickstart/sock5.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+sudo apt install pkg-config build-essential libssl-dev curl jq
+```
+
+- `Rust & cargo >= v1.56`
+
+We recommend using the [Rust shell script installer](https://www.rust-lang.org/tools/install). Installing cargo from your package manager (e.g. `apt`) is not recommended as the packaged versions are usually too old.
+
+If you really don't want to use the shell script installer, the [Rust installation docs](https://forge.rust-lang.org/infra/other-installation-methods.html) contain instructions for many platforms.
+
+### Download and build Nym binaries
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/sock5.md]: Check the formatting in the other pages and change this please - refer to the 'building nym' page.

File: versioned_docs/version-stable/quickstart/sock5.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+        2022-04-27T16:15:51.706Z INFO  nym_socks5_client::client > Starting loop cover traffic stream...
+        2022-04-27T16:15:51.707Z INFO  nym_socks5_client::client > Starting socks5 listener...
+        2022-04-27T16:15:51.707Z INFO  nym_socks5_client::socks::server > Listening on 127.0.0.1:1080
+        2022-04-27T16:15:51.707Z INFO  nym_socks5_client::client        > Client startup finished!
+        2022-04-27T16:15:51.707Z INFO  nym_socks5_client::client        > The address of this client is: BFKhbyNsSVwbsGSLwHDkfwH5mwZqZYpnpNjjV7Xo25Xc.EFWd1geWspzyVeinwXrY5fCBMRtAKV1QmK1CNFhAA8VG@BNjYZPxzcJwczXHHgBxCAyVJKxN6LPteDRrKapxWmexv
+        2022-04-27T16:15:51.707Z INFO  nym_socks5_client::socks::server > Serving Connections...
+
+</details>
+
+![Nym connect app](/img/docs/keybase-settings.gif)
```
Comment (mfahampshire) [inline versioned_docs/version-stable/quickstart/sock5.md]: Why is this here?

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: versioned_docs/version-stable/quickstart/sock5.md (hunk 10 above)
Comment (iamonuwa) [reply versioned_docs/version-stable/quickstart/sock5.md]: Removing now

File: versioned_docs/version-stable/quickstart/nym-connect.md (hunk 8 above)
Comment (iamonuwa) [reply versioned_docs/version-stable/quickstart/nym-connect.md]: I'm on mac and tested it on mac. Works

File: versioned_docs/version-stable/quickstart/nym-connect.md (hunk 7 above)
Comment (iamonuwa) [reply versioned_docs/version-stable/quickstart/nym-connect.md]: I get this issue on mac also. Happens to any app download from the internet

**Comment by mfahampshire** [comment]
Changes LGTM. 

I'm not going to merge this, as I think we can actually use this branch as the jump-off point for a bigger documentation update that the quickstart is the first part of. 

I'm going to move it to draft now and we can keep adding to it.

**Pull request merged**